CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Go Buildpack Changelog
 
 ## Unreleased
-* Add go1.16.8, use for go1.16
-* Add go1.17.1, use for go1.17
+* Add go1.16.9, use for go1.16
+* Add go1.17.2, use for go1.17
+* Add go1.16.8
+* Add go1.17.1
```

Comment:
It looks like the CHANGELOG wasn't updated (by us) when the buildpack was published for #474:
```
$ h buildpacks:versions heroku/go | head -n4
Version  Released At               Status
───────  ────────────────────────  ─────────
155      2021-09-13T16:18:38.053Z  published
154      2021-08-18T21:18:15.534Z  published
```

As such, let's fix that here :-)

```suggestion
* Add go1.16.9, use for go1.16
* Add go1.17.2, use for go1.17

## v155 (2021-09-13)
* Add go1.16.8, use for go1.16
* Add go1.17.1, use for go1.17
```